test/e2e/report_dynamic_input_data_test.go

```diff
@@ -291,4 +292,19 @@ func testReportIsNotDeletedWhenReportDependsOnIt(t *testing.T, testReportingFram
 		"timed out waiting"),
 		"should have timed out waiting on subReport expiration as subReport should not have been deleted",
 	)
+	listOptions := metav1.ListOptions{}
```

Comment:
```suggestion
	listOptions := metav1.ListOptions{
	    FieldSelector: "involvedObject.Name="subreport-should-not-delete,type=Warning"
	```
...
require.NoError(t, err, "failed to list the events in the test namespace")
require.GreaterOrEqual(len(events.Items), 1, "expected the list of events matching the %s field selector to be greater than zero")
```
```

Reply:
yep, better, trying now.

Reply:
```                	Error:      	Received unexpected error:
                	            	field label not supported: involvedObject.Name```

Reply:
Going to leave as is, there are not many events to loop through.